### DISCUSSION TITLE
Issue in Ubuntu 21.10

### DISCUSSION THREAD

**Discussion by vashero** [Chia-Network/chia-blockchain#8334] [Support]
When I try to Open the installed GUI 1.2.5, I get a processing mouse cursor... then Nothing.... 
I also reinstalled and same problem. .... When I install via terminal and try to run..... I get the following:

```
> chia-blockchain@1.2.5 electron
> electron .

/home/sgoff/chia-blockchain/app.asar.unpacked/daemon
false
startUrl file:///home/sgoff/chia-blockchain/chia-blockchain-gui/build/renderer/index.html
/home/sgoff/chia-blockchain/app.asar.unpacked/daemon
false
/home/sgoff/chia-blockchain/app.asar.unpacked/daemon
false
Running python script
Script ../chia/daemon/server.py
child process success
(node:17096) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
[17096:0904/205050.281226:FATAL:gpu_data_manager_impl_private.cc(445)] GPU process isn't usable. Goodbye.
/home/sgoff/chia-blockchain/chia-blockchain-gui/node_modules/electron/dist/electron exited with signal SIGTRAP
```
And then Nothing happens.  Any help would be greatly appreciated... as I have 2K plots on this machine and would like to get them back up and going.

**Comment by loppefaaret** [comment]
are you doing `npm run electron` from `/chia-blockchain/chia-blockchain-gui/` ?

**Reply by vashero** [reply]
Yes, I have also tried installing and running GUI from deb package.  This problem seemed to occur after a recent ubuntu update/upgrade

**Reply by loppefaaret** [reply]
i dont know what i was looking at last time i looked at this, but it is clearly showing that it is an GPU issue - do you have for instance some NVIDIA drivers installed for a GPU ? perhaps try and unload any GPU drivers, and revert back to default ubuntu provide modules ?

**Reply by vashero** [reply]
Ill check this and let you know.  I may end up just upgrading the video card in it... as I don't do anything but chia on this machine, so I threw a Nvidia GTX 710 in it.   It worked fine in the past, ill try to revert any updates to this driver.

**Reply by vashero** [reply]
That fixed it, Thanks!

**Comment by BrandtH22** [comment]
@vashero , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
